skbio/diversity/_driver.py

```diff
@@ -396,6 +396,9 @@ def beta_diversity(metric, counts, ids=None, validate=True, pairwise_func=None,
     sklearn.metrics.pairwise_distances
 
     """
+    if isinstance(counts, pd.core.frame.DataFrame) and ids is None:
```

Comment:
```suggestion
    if isinstance(counts, pd.DataFrame) and ids is None:
```

...I think that should work given:

```In [1]: import pandas as pd

In [2]: d = pd.DataFrame([])

In [3]: isinstance(d, pd.DataFrame)
Out[3]: True
```